docs/mediterranean.html

```diff
@@ -0,0 +1,241 @@
+  <meta name="description" content="">
+  <meta name="author" content="">
+
+  <title>TOPIOS: Tracking Of Plastic In Our Seas</title>
+
+  <!-- Bootstrap core CSS -->
+  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Font Awesome icons -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/academicons.css"/>
+
+  <!-- Custom styles for this template -->
+  <link href="css/modern-business.css" rel="stylesheet">
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-111703388-1"></script>
+  <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-111703388-1');
+  </script>
+</head>
+
+<body>
+
+<!-- Navigation -->
+<!--suppress SpellCheckingInspection -->
+<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark fixed-top">
+  <div class="container">
+    <a class="navbar-brand" href="index.html">TOPIOS: Tracking Of Plastic In Our Seas</a>
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+  </div>
+</nav>
+
+
+<!-- Page Content -->
+<div class="container">
+
+  <hr>
+
+  <h1 class="display-3" style="text-align: center;">Closing the Mediterranean Floating Plastic Mass Budget</h1>
+	
+	<p>
+	It is currently not well known how much plastics are entering our oceans, and what happens with these plastics in the environment. Most plastics are lighter than water, and are therefore expected to float. However, over time a layer of algae increases the weight of these plastic particles, possibly leading to these particles sinking down. Another significant part of plastics might eventually end up on our coastlines.
+	</p><p>
+	Here, we developed a method to estimate the total input of plastics in the Mediterranean, and where these plastics are expected to end up. The Mediterranean is a nice test case: it is confined, a lot of people are living in this area, which likely means a lot of plastic pollution as well, and there are many plastic concentration measurements available from the surface water.
+    </p><p>
+    A model of floating particles is made to represent floating plastics as accurately as possible, by comparing it with these plastic concentration measurements. Results from the model show that the total plastic input into the Mediterranean sea likely ranges between 2,200-4,000 tonnes for the year 2015, that only a small part of the plastics remain afloat (about 240-390 tonnes), that about half of the plastics start sinking down, and that about half of the plastics are expected to end up on the coastlines.
```

Comment:
```suggestion
    A model of floating particles is made to represent floating plastics as accurately as possible, by comparing it with these plastic concentration measurements. Results from the model show that the total plastic input into the Mediterranean Sea likely ranges between 2,200-4,000 tonnes for the year 2015, that only a small part of the plastics remain afloat (about 240-390 tonnes), that about half of the plastics start sinking down, and that about half of the plastics are expected to end up on the coastlines.
```
Use capital S for Mediterranean Sea